rpcs3/Emu/Cell/Modules/cellSearch.cpp

```diff
@@ -340,6 +922,37 @@ error_code cellSearchGetContentInfoPath(vm::cptr<CellSearchContentId> contentId,
 		return CELL_SEARCH_ERROR_PARAM;
 	}
 
+	switch(g_fxo->get<search_info>()->state)
+	{
+		case SearchState::IDLE:
+			break;
+		case SearchState::NOT_INITIALIZED:
+		case SearchState::INITIALIZING:
+			return CELL_SEARCH_ERROR_NOT_INITIALIZED;
+		case SearchState::FINALIZING:
+			return CELL_SEARCH_ERROR_FINALIZING;
+		case SearchState::IN_PROGRESS:
+		case SearchState::CANCELING:
+			return CELL_SEARCH_ERROR_BUSY;
+		default:
+			return CELL_SEARCH_ERROR_GENERIC;
+	}
+
+	u64 id = *reinterpret_cast<const u64*>(contentId->data);
```

Comment:
Const

Reply:
use std::bit_cast maybe? we implement it.